style/hudl_sql_style_guide.md

```diff
@@ -5,33 +5,101 @@ description: A guide to writing clean, clear, and consistent SQL.
 ---
 
 # Purpose
-
+This document exists as a resource of guiding principles for those who build, refactor and work with data models. It contains Decision Science’s requirements and best practices for data modeling, SQL style and data testing.
 
 # Principles
-
-* We take a disciplined and practical approach to writing code.
+* We take a disciplined and practical approach to writing code
 * We regularly check-in code to Github
-* We believe consistency in style is very important.
-* We demonstrate intent explicitly in code, via clear structure and comments where needed.
-
-# Rules
-
-## General
-
-* No tabs. 2 spaces per indent.
-* No trailing whitespace.
-* 80 character limit per line
-* Include links to hudl id constants or references to important links
-* Always use AS when creating aliases for columns, tables, subqueries
-* Always capitalize SQL keywords (e.g., `SELECT` or `AS`)
-* Always capitalize SQL datatypes (e.g., `VARCHAR` or `DATE`)
-* Comments should go near the top of your query, or at least near the closest `SELECT`
-* Try to only comment on things that aren't obvious about the query (e.g., why a particular ID is hardcoded, etc.)
-* All dates should be formatted as 'YYYY-MM-DD'
-* Use `!=` for the inequality operator
-
-## Naming Conventions
-
+* We believe consistency in style is very important
+* We demonstrate intent explicitly in code, via clear structure and comments where needed
+* We value DRY (do not repeat yourself) code and readability, but will favor readability over DRY when the two are in conflict
```

Comment:
I'm not sure I know how to reconcile these two ideas

Reply:
I agree with Tyler and I added a comment above about descriptive naming. Part of making DRY code readable is naming the reusable components in a more descriptive manner.